chainer/function_hook.py

```diff
@@ -76,15 +84,24 @@ class FunctionHook(object):
        as a thread local object. So, function hooks registered
        are different depending on threads.
 
+    If the hook is registered in this way, :meth:`~chainer.FunctionHook.added`
+    and :meth:`~chainer.FunctionHook.deleted` are given ``None`` as the
+    ``function`` argument.
```

Comment:
How about rephrasing (matching with the below one):
None is passed to the function argument of added and deleted.

Reply:
Fixed using "passed as".

Reply:
Ah, yes you are right. I was confused with "pass to function as an argument".